docs/graphql/manual/deployment/digitalocean/index.rst

```diff
@@ -0,0 +1,76 @@
+Run Hasura GraphQL Engine using Docker on DigitalOcean
+======================================================
+
+This guide helps you set up the Hasura GraphQL engine using Docker on DigitalOcean.
+
+**Prerequisites**:
+
+- No Special requirement, just putty if you are on windows
+
+Step 1: Create and prepare a droplet
+------------------------------------
+
+- Login to your DigitalOcean account
+- Click on create droplet then on ``One-click apps``
+- Choose ``Docker`` from the ``One-click apps`` list
+- Choose your droplet size
+- Choose nearest datacenter region from your location
+- Select your ssh key add if you haven't added
+- Choose a hostname and then click on create button.
+- Now SSH into the droplet.
+- Run ``apt install update && apt upgrade -y`` to update system 
+
+Step 2: Get the Docker deployment and Caddy file 
+------------------------------------------------
+
+The `hasura/graphql-engine/install-manifests/docker-compose-https <https://github.com/hasura/graphql-engine/tree/master/install-manifests/docker-compose-https>`_ repo
+contains all installation manifests required to deploy Hasura on DigitalOcean.
+
+Get the docker deployment and Caddy file from there:
+
+.. code-block:: bash
+
+   $ wget https://raw.githubusercontent.com/hasura/graphql-engine/master/install-manifests/docker-compose-https/docker-compose.yaml
+   $ wget https://raw.githubusercontent.com/hasura/graphql-engine/master/install-manifests/docker-compose-https/Caddyfile
+
+Step 3: Set an access key and point domain to the public ip of the droplet. 
```

Comment:
This step can be an optional one for setting up HTTPS. We can word it such that a user who doesn't need HTTPS can skip this.

Adding access key can be moved to a step towards the end under section `Securing the GraphQL Endpoint`

Reply:
then the user needs to download another .yaml file, isn't it?

Reply:
Yes, if the user doesn't need HTTPS, they can just download https://github.com/hasura/graphql-engine/blob/master/install-manifests/docker-compose/docker-compose.yaml

Maybe you can have 2 sub-sections: with/without HTTPS.

Reply:
got it. 👍